network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -390,17 +393,39 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         ),
     ]
 
+    @property
+    def product_type(self):
+        if isinstance(self, SoftwareProductPage):
+            return "software"
+        elif isinstance(self, GeneralProductPage):
+            return "general"
+        else:
+            return "standard"
```

Comment:
let's make this `unknown` because there is no such thing a "not a general or software product" right now.